gms/feature_service.hh

```diff
@@ -178,6 +179,10 @@ public:
         return bool(_uda);
     }
 
+    bool cluster_supports_reduced_revenced_queries_limit() const {
```

Comment:
Please add here a short description of this feature as a comment.

Reply:
BTW maybe a better name for this feature is `SEPARATE_PAGE_SIZE_AND_SAFETY_LIMIT` or something similar. The change although motivated by the problem with reverse queries helps us more generally with a clear separation of page size into a field of its own, resolving possible confusion and helping cases where both he limit and the page size is needed.
But I don't want to bike-shed on the name, take this as a suggestion only.

Reply:
I have not much preference regarding the name so I can as well change it if that's what you desire. I will add a comment too.